source/Calamari.Common/Plumbing/CalamariEnvironment.cs

```diff
@@ -12,6 +12,9 @@ public static class CalamariEnvironment
         /// </summary>
         public static readonly bool IsRunningOnMono = Type.GetType("Mono.Runtime") != null;
 
+        public static bool IsRunningOnKubernetes =>
+            Environment.GetEnvironmentVariable(KubernetesEnvironmentVariables.KubernetesAgentNamespace) != null;
```

Comment:
The more I think about it, `string.IsNullOrWhitespace` is safer
```suggestion
            !string.IsNullOrWhiteSpace(Environment.GetEnvironmentVariable(KubernetesEnvironmentVariables.KubernetesAgentNamespace));
```